myapp/app/views/tasks/index.html.erb

```diff
@@ -3,28 +3,55 @@
 
   <%= link_to 'New Task', new_task_path, class: 'btn btn-primary mb-3' %>
 
+  <div class="row mb-3">
+    <div class="col-md-6">
+      <%= form_tag(tasks_path, method: :get, class: "search-form") do %>
+        <div class="row">
+          <div class="col-md-6">
+            <div class="form-group">
+              <%= label_tag :title, t('forms.title') %>
+              <%= text_field_tag :title, params[:title], class: "form-control" %>
+            </div>
+          </div>
+          <div class="col-md-6">
+            <div class="form-group">
+              <%= label_tag :status, t('forms.status') %>
+              <%= select_tag :status, options_for_select(Task.statuses.keys.map { |key| [t("statuses.#{key}"), key] }, selected: params[:status]), include_blank: true, class: "form-control" %>
+            </div>
+          </div>
+        </div>
+        <div class="form-group mt-2">
+        <%= submit_tag "Search", class: "btn btn-primary" %>
+        </div>
+      <% end %>
+    </div>
+  </div>
+
+  
+
   <div class="mb-3">
     <%= link_to 'Sort by Created At (Asc)', tasks_path(sort_by: 'created_at', sort_direction: 'asc'), class: 'btn btn-primary me-2' %>
     <%= link_to 'Sort by Created At (Desc)', tasks_path(sort_by: 'created_at', sort_direction: 'desc'), class: 'btn btn-primary me-2' %>
     <%= link_to 'Sort by Deadline (Asc)', tasks_path(sort_by: 'deadline', sort_direction: 'asc'), class: 'btn btn-primary me-2' %>
     <%= link_to 'Sort by Deadline (Desc)', tasks_path(sort_by: 'deadline', sort_direction: 'desc'), class: 'btn btn-primary' %>
   </div>
 
-
+  
   <% if @tasks.present? %>
     <div class="row">
       <% @tasks.each do |task| %>
         <div class="col-md-3"> <!-- Adjusting the height of columns -->
           <div class="card mb-4" style="height: 250px;">
             <div class="card-body d-flex flex-column">
-              <h5 class="card-title">Title: <%= link_to task.title, task %></h5> <!-- Add a link to the show page -->
+            <h5 class="card-title">Title: <%= link_to task.title, task %></h5> <!-- Add a link to the show page -->
```

Comment:
Why did you delete indent?

Reply:
Maybe I ran rubocop and it was automatically modified